docs/source/openstack.rst

```diff
@@ -65,23 +65,23 @@ os_obj
 
 OpenStack Object Storage can be provisioned using this resource.
 
-* :docs1.5:`Topology Example <workspaces/topologies/os-obj-new.yml>`
+* :docs1.5:`Topology Example <workspace/topologies/os-obj-new.yml>`
```

Comment:
We're trying to do away with the `workspace` directory.  Can you link to `workspaces/openstack/topologies/os-obj-new.yml`?  And also do the same with the other changes here?

Reply:
Done, there was no os_keypair.yml in the new location, so I just copied it from old directory, I hope it is ok.